library/ortools/java/deps.bzl

```diff
@@ -32,21 +32,24 @@ def google_or_tools_mac():
         name = "google_or_tools_darwin",
         urls = ["https://github.com/google/or-tools/releases/download/v8.0/or-tools_MacOsX-10.15.7_v" + version + ".tar.gz"],
         strip_prefix = "or-tools_MacOsX-10.15.7_v" + version + "",
-        build_file_content = google_or_tools_export_files(version, "darwin")
+        build_file_content = google_or_tools_export_files(version, "darwin"),
+        sha256 = "3c921c16c8162b337744dc45819dd5cf0c7a43382615edb8f599320bcf31773a"
```

Comment:
We didn't have `sha256` for these, and we were downloading them via a GitHub /releases URL. This increases hermetic-ness, so I've added this for all of the `google_or_tools_*`.